app/webpack.config.js

```diff
@@ -21,7 +21,10 @@ module.exports = {
             ]),
         new webpack.DefinePlugin({
                 'process.env.NODE_ENV': JSON.stringify('development')
-            })
+            }),
+        // the ajv library really does not play well with webpack.
+        // This forces it to play a little nicer.
+        new webpack.IgnorePlugin(/\.(jst|md|def|d\.ts)/, /ajv/)
```

Comment:
sweet! I was in the process of messing with this library but looks like this does it